wakuv2/waku.go

```diff
@@ -1275,16 +1279,27 @@ func (w *Waku) Stop() error {
 
 	w.node.Stop()
 
+	time.Sleep(2 * time.Second)
+	w.logger.Info("<<< push to telemetry client")
+
+	if w.statusTelemetryClient != nil {
+		w.statusTelemetryClient.PushPeerCount(w.PeerCount())
+	}
+
 	if w.protectedTopicStore != nil {
 		err := w.protectedTopicStore.Close()
 		if err != nil {
 			return err
 		}
 	}
 
+	w.logger.Info("<<< wait for wait group")
```

Comment:
And maybe this should be removed as well, (and also line 1301)

Reply:
452fb5e078dec81bdaf2460eb292b92f3bde956c